frontend/app/dashboard/template.hbs

```diff
@@ -0,0 +1,222 @@
+<h1>{{t 'index.greeting'}}</h1>
+
+<section>
+  <div>
+    <div uk-grid class="uk-child-width-1-1 uk-child-width-1-2@m" >
+      <div uk-grid class="uk-child-width-1-1" >
+
+      <div class="uk-padding uk-padding-remove-bottom">
+        {{#uk-card hover=true as |card|}}
+          {{#card.header}}
+          {{#card.title}}{{t 'nav.timed'}}{{/card.title}}
+          {{/card.header}}
+          {{#card.body}}
+            <div class="uk-child-width-1-1  uk-child-width-1-2@m  uk-text-center uk-grid-divider" uk-grid >
+              {{#each projects as |p index|}}
+              <div>
+                <div class="uk-flex uk-flex-column uk-flex-between uk-height-1-1	" data-test-project={{p.id}}>
+                  <div>
+                    {{#if (await session.user)}}
+                      {{#link-to (concat (if (or (get (await session.user) 'adsyUser') (get (await session.user) 'admin')) "timed-admin" "timed-subscriptions") ".detail") p.id class="uk-link-text" }}
+                        <p class="uk-text-lead" data-test-project-name>{{p.name}}</p>
+                      {{/link-to}}
+                    {{/if}}
+                    <p data-test-project-total>{{format-duration p.totalTime}}{{t 'dashboard.remaining'}}</p>
+                  </div>
+                  {{svg-battery percentage=p.percentage data-test-percentage=true}}
```

Comment:
`data-test-percentage=true` what should this do?

Reply:
Couldn't you just pass `data-test-percentage`?

Reply:
no this causes a `Expecting 'EQUALS', got 'CLOSE'` exception while compiling.

Reply:
Did you pass it as the first argument? See https://github.com/simplabs/ember-test-selectors#usage-in-components for reference..